packages/sonarwhal/package.json

```diff
@@ -130,7 +131,7 @@
     "build:ts": "tsc",
     "clean": "rimraf dist",
     "lint": "npm-run-all lint:*",
-    "lint:js": "eslint . --cache --report-unused-disable-directives --ext js --ext md --ext ts",
+    "lint:js": "eslint . --cache --report-unused-disable-directives --ext js --ext md --ext ts --ignore-pattern dist/**/*.js --ignore-pattern dist/**/*.d.ts",
```

Comment:
I don't think this line should be changed. It's working fine in the other packages (at least now, don't know about earlier in the refactor)